pkg/plugins/generic/generic_plugin.go

```diff
@@ -87,12 +87,31 @@ func (p *GenericPlugin) Apply() error {
 			return nil
 		}
 	}
+
+	// Create a map with all the PFs we will need to configure
+	// we need to create it here before we access the host file system using the chroot function
+	// because the SkipConfigVf needs the mstconfig package that exist only inside the sriov-config-daemon file system
+	vfsToSkip := map[string]bool{}
```

Comment:
nit: it would be good to move this code into the separate function

Reply:
done